recipes/coverage-lcov/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "coverage-lcov" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3de40d620cb13e43c746126be3154252d1296787a9f2a54b14683763cf9c0c46
+
+build:
+  skip: true  # [win]
```

Comment:
Can this not be `noarch`? If one of the dependencies is missing that's fine.

In that case, the entry point needs to be added as well.